pom.xml

```diff
@@ -120,7 +120,7 @@
     <plugin.nexus-maven-staging.version>1.6.7</plugin.nexus-maven-staging.version>
     <plugin.sonar-maven.version>3.9.1.2184</plugin.sonar-maven.version>
     <plugin.sortpom-maven.version>2.15.0</plugin.sortpom-maven.version>
-    <plugin.versions-maven.version>2.14.2</plugin.versions-maven.version>
+    <plugin.versions-maven.version>2.17.1</plugin.versions-maven.version>
```

Comment:
Versions plugin 2.17.+ requires 3.6.3+ -- we should update the `maven.min-version` as well

See: https://github.com/mojohaus/versions/releases/tag/2.17.0